tsconfig.json

```diff
@@ -44,7 +44,7 @@
 
     /* Module Resolution Options */
     "moduleResolution": "node",                     /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
-    // "baseUrl": "./",                             /* Base directory to resolve non-absolute module names. */
+    "baseUrl": "src",                               /* Base directory to resolve non-absolute module names. */
```

Comment:
Very useful :+1: :slightly_smiling_face:

Reply:
Yes, this is really great for shorting of imports everywhere. :wink: